src/main/java/edu/kis/powp/jobs2d/features/DrawerPanelClickMouseListenerFeature.java

```diff
@@ -0,0 +1,61 @@
+package edu.kis.powp.jobs2d.features;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+
+public class DrawerPanelClickMouseListenerFeature implements MouseListener, Job2dDriver {
```

Comment:
This class shouldn't be a driver, since it is not (and do not need to be) used as a driver.